### PR TITLE
Introduction of LocalizedValueDescriptor

### PR DESCRIPTION
**Pull request by seroy**
no need inheritance from LocalizedModel anymore

### CODE REVIEW

File: localized_fields/fields/localized_field.py [hunk 1]
```diff
@@ -1,12 +1,70 @@
...
+        # This is slightly complicated, so worth an explanation.
+        # `instance.localizedvalue` needs to ultimately return some instance of
+        # `LocalizedValue`, probably a subclass.
+
+        # The instance dict contains whatever was originally assigned
+        # in __set__.
+        if self.field.name in instance.__dict__:
+            value = instance.__dict__[self.field.name]
+        else:
+            instance.refresh_from_db(fields=[self.field.name])
```
Comment (Photonios) [inline localized_fields/fields/localized_field.py]: Why does the value have to be refreshed?

**Comment by Photonios** [comment]
Please keep the `LocalizedModel` field around for backwards compatibility.

**Comment by si14** [comment]
Hi there!

We are finally preparing a bunch of separate pull requests per your recommendation in #1. This one is the first one with an introduction of `LocalizedValueDescriptor`. Would love your thoughts on it!

**Comment by si14** [comment]
Ooops, you are already on it. Thanks for a rapid response :)

File: localized_fields/fields/localized_field.py (hunk 1 above)
Comment (seroy) [inline localized_fields/fields/localized_field.py]: If the value of the field was not retrieved from database due to the use of the defer() or only() methods, we need to refresh from db

**Comment by si14** [comment]
Hi @Photonios!

Do you think it's OK with the last addition? My colleague @seroy added `LocalizedModel` back.

**Comment by Photonios** [comment]
@si14 

This looks great! I'll have to verify it locally to make sure everything works properly, but I don't see much to comment on :)

**Comment by si14** [comment]
Hey @Photonios, did you have a chance to check it out? It seems that another PR was merged in the meantime so there are some conflicts now, but we can sort it out if you think it's good to go.

I'm sorry for being a bit pushy, we just have another batch of PRs incoming (including translated FileField!), but they are implemented on top of this one :)

**Comment by si14** [comment]
Hi folks!

Any news on this PR?

**Comment by Photonios** [comment]
Hi!

Sorry, this looks great. Let me attempt to merge this :-)

**Pull request merged**